SwiftzTests/StreamSpec.swift

```diff
@@ -149,5 +149,12 @@ class StreamSpec : XCTestCase {
 				}
 			}
 		}
-	}
+
+        property("sequence occurs in order") <- forAll { (xs : [String]) in
+            let seq = sequence(xs.map({ x in Stream.pure(x).take(1) }))
```

Comment:
What does limiting the size of generated lists do (either with custom arguments or `Gen.fromElementsIn(1...10)`, etc.)?